ui/core/_core-bs3.scss

```diff
@@ -108,6 +108,13 @@
       }
     }
   }
+
+  .slds-modal__header,
+  .slds-notify--toast {
+    h2 {
+      margin-top: $spacing-none;
+    }
+  }
```

Comment:
Instead of `slds-`, please use `#{$css-prefix}`.

Also, should this change apply to all the header tags?